src/components/rangeslider/draw.js

```diff
@@ -455,7 +455,8 @@ function drawRangePlot(rangeSlider, gd, axisOpts, opts) {
             id: id,
             plotgroup: plotgroup,
             xaxis: xa,
-            yaxis: ya
+            yaxis: ya,
+            isRangePlot: true
```

Comment:
I don't think this range-plot range slider fix got :lock: anywhere. We could add a case in `select_test.js` selects a point in the main plot and asserts that it is really that main-plot plot that gets the selected style.

Reply:
Ah, thanks for reminding me about the rangeslider + selection fix! 🔒-> https://github.com/plotly/plotly.js/pull/2561/commits/c77a8a319ccfc6aeff044e48cee561e281dec21a